### PR TITLE
Update tf.nn.softmax_cross_entropy_with_logits to v2

### PR DESCRIPTION
**Pull request by nealwu**

### CODE REVIEW

File: dual_net.py [hunk 1]
```diff
@@ -206,8 +206,8 @@ def my_res_layer(inputs):
     # train ops
     global_step = tf.train.get_or_create_global_step()
     policy_cost = tf.reduce_mean(
-        tf.nn.softmax_cross_entropy_with_logits(
-            logits=logits, labels=labels['pi_tensor']))
+        tf.nn.softmax_cross_entropy_with_logits_v2(
+            logits=logits, labels=tf.stop_gradient(labels['pi_tensor'])))
```
Comment (brilee) [inline dual_net.py]: curious why this is necessary

Comment (nealwu) [reply dual_net.py]: Ah, explained this on the other PR https://github.com/tensorflow/minigo/pull/139#discussion_r177646634 but forgot to copy it over:

This `tf.stop_gradient` may not be necessary, but I added it just to be safe.

See https://www.tensorflow.org/api_docs/python/tf/nn/softmax_cross_entropy_with_logits and https://www.tensorflow.org/api_docs/python/tf/nn/softmax_cross_entropy_with_logits_v2 for context.

**Comment by brilee** [comment]
Ping - merge this?

**Comment by nealwu** [comment]
Thanks @brilee. FYI I don't have the ability to merge branches in this repo. Same situation on https://github.com/tensorflow/minigo/pull/139

**Comment by amj** [comment]
@brilee  Did this get reverted?  I see it says 'merged' but the v2 version is not present at HEAD (https://github.com/tensorflow/minigo/blob/master/dual_net.py#L209 )

**Pull request merged**